packages/atomic/src/components/facets-v1/atomic-color-facet/atomic-color-facet.tsx

```diff
@@ -222,6 +233,10 @@ export class AtomicColorFacet
             onClick={onClick}
             searchQuery={this.facetState.facetSearch.query}
           >
+            <div
+              part={`value-${displayValue}`}
```

Comment:
I would strip the display value from everything but hyphens, underscores, letters, or numbers `_a-zA-Z0-9-`
https://stackoverflow.com/a/449000